source/fuzz/repeated_pass_recommender_standard.cpp

```diff
@@ -0,0 +1,325 @@
+    // - The guard of the dead continue is a good candidate for obfuscation
+    return RandomOrderAndNonNull({pass_instances_->GetObfuscateConstants()});
+  }
+  if (&pass == pass_instances_->GetAddEquationInstructions()) {
+    // - Equation instructions can create synonyms, which we can apply
+    // - Equation instructions collaborate with one another to make synonyms, so
+    //   having added some it is worth adding more
+    return RandomOrderAndNonNull(
+        {pass_instances_->GetApplyIdSynonyms(),
+         pass_instances_->GetAddEquationInstructions()});
+  }
+  if (&pass == pass_instances_->GetAddFunctionCalls()) {
+    // - Called functions can be inlined
+    // - Irrelevant ids are created, so they can be replaced
+    return RandomOrderAndNonNull({pass_instances_->GetInlineFunctions(),
+                                  pass_instances_->GetReplaceIrrelevantIds()});
+  }
+  if (&pass == pass_instances_->GetAddGlobalVariables()) {
+    // - New globals provide new possibilities for making access chains
+    // - We can load from and store to new globals
+    return RandomOrderAndNonNull({pass_instances_->GetAddAccessChains(),
+                                  pass_instances_->GetAddLoads(),
+                                  pass_instances_->GetAddStores()});
+  }
+  if (&pass == pass_instances_->GetAddImageSampleUnusedComponents()) {
+    // - This introduces an unused component whose id is irrelevant and can be
+    //   replaced
+    return RandomOrderAndNonNull({pass_instances_->GetReplaceIrrelevantIds()});
+  }
+  if (&pass == pass_instances_->GetAddLoads()) {
+    // - Loads might end up with corresponding stores, so that pairs can be
+    //   replaced with memory copies
+    return RandomOrderAndNonNull(
+        {pass_instances_->GetReplaceLoadsStoresWithCopyMemories()});
+  }
+  if (&pass == pass_instances_->GetAddLocalVariables()) {
+    // - New locals provide new possibilities for making access chains
+    // - We can load from and store to new locals
+    return RandomOrderAndNonNull({pass_instances_->GetAddAccessChains(),
+                                  pass_instances_->GetAddLoads(),
+                                  pass_instances_->GetAddStores()});
+  }
+  if (&pass == pass_instances_->GetAddLoopPreheaders()) {
+    // - No obvious follow-on passes
+    return {};
+  }
+  if (&pass == pass_instances_->GetAddOpPhiSynonyms()) {
+    // - New synonyms can be applied
+    // - If OpPhi synonyms are introduced for blocks with dead predecessors, the
+    //   values consumed from dead predecessors can be replaced
+    return RandomOrderAndNonNull(
+        {pass_instances_->GetApplyIdSynonyms(),
+         pass_instances_->GetReplaceOpPhiIdsFromDeadPredecessors()});
+  }
+  if (&pass == pass_instances_->GetAddParameters()) {
+    // - This introduces irrelevant ids, which can be replaced
+    return RandomOrderAndNonNull({pass_instances_->GetReplaceIrrelevantIds()});
```

Comment:
We could probably add `FuzzerPassAddSynonyms` to create interesting usages of new parameters.

Reply:
Done.